types/fast-json-stable-stringify/index.d.ts

```diff
@@ -1,22 +0,0 @@
-// Type definitions for fast-json-stable-stringify 2.0
-// Project: https://github.com/epoberezkin/fast-json-stable-stringify
-// Definitions by: BendingBender <https://github.com/BendingBender>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
-export = stringify;
-
-declare function stringify(obj: any, options?: stringify.Options | stringify.Comparator): string;
```

Comment:
`options` is not typed in the new typings provided.

Reply:
Made a PR: https://github.com/epoberezkin/fast-json-stable-stringify/pull/66